### PR TITLE
[Bugfix] ControlNetXS for peft v0.7.0

### PR DESCRIPTION
**Pull request by okotaku**
# What does this PR do?

When installing peft>=0.7.0, following error raised.

```py
from diffusers import ControlNetXSModel, UNet2DConditionModel

unet = UNet2DConditionModel.from_pretrained(
    "stabilityai/stable-diffusion-xl-base-1.0", subfolder="unet")
cn = ControlNetXSModel.init_original(
    unet, is_sdxl=True)
```

```bash
Traceback (most recent call last):
  File "/workspace/d.py", line 5, in <module>
    cn = ControlNetXSModel.init_original(
  File "/workspace/src/diffusers/models/controlnetxs.py", line 162, in init_original
    return ControlNetXSModel.from_unet(
  File "/workspace/src/diffusers/models/controlnetxs.py", line 488, in from_unet
    return cls(**kwargs)
  File "/workspace/src/diffusers/configuration_utils.py", line 649, in inner_init
    init(self, *args, **init_kwargs)
  File "/workspace/src/diffusers/models/controlnetxs.py", line 307, in __init__
    increase_block_input_in_encoder_resnet(
  File "/workspace/src/diffusers/models/controlnetxs.py", line 824, in increase_block_input_in_encoder_resnet
    assert hasattr(old_conv1, a)
AssertionError
```

I solved it.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/diffusers/models/controlnetxs.py [hunk 1]
```diff
@@ -817,9 +817,7 @@ def increase_block_input_in_encoder_resnet(unet: UNet2DConditionModel, block_no,
     norm_kwargs = {a: getattr(old_norm1, a) for a in norm_args}
     norm_kwargs["num_channels"] += by  # surgery done here
     # conv1
-    conv1_args = (
-        "in_channels out_channels kernel_size stride padding dilation groups bias padding_mode lora_layer".split(" ")
-    )
+    conv1_args = "in_channels out_channels kernel_size stride padding dilation groups bias padding_mode".split(" ")
```
Comment (sayakpaul) [inline src/diffusers/models/controlnetxs.py:820]: Can we condition this by making use of `USE_PEFT_BACKEND`? 

https://github.com/huggingface/diffusers/blob/664e931bcbddd47c21c3d4d547221506f588b9e9/src/diffusers/loaders/lora.py#L414

**Review by patrickvonplaten — APPROVED** [review]
Looks good to me

**Comment by sayakpaul** [comment]
@okotaku ping me once the CI is complete. I will merge :)

**Comment by okotaku** [comment]
This bug is fixed here.
https://github.com/huggingface/diffusers/pull/6228/files

I will close it.

**Comment by sayakpaul** [comment]
Okay! Cc: @DN6.

**Pull request closed without merging**